node/node.go

```diff
@@ -80,12 +85,22 @@ func (n *Node) Start(ctx context.Context) error {
 		log.Errorf("starting %s Node: %s", n.Type, err)
 		return fmt.Errorf("node: failed to start: %w", err)
 	}
-	log.Infof("started %s Node", n.Type)
+
+	// start server only if Full node // TODO @renaynay: eventually we'll add the RPC server to the light node
+	if n.Type == Full {
```

Comment:
Not urgent but IMO we should consider making the RPC server optional, even for full nodes.

Reply:
Yes i made a comment in the code that we'll eventually be able to toggle it on and off (and during runtime)